packages/boxel-ui/addon/src/components/resizable-panel-group/index.gts

```diff
@@ -330,23 +370,24 @@ export default class ResizablePanelGroup extends Component<Signature> {
           ? nextElLength -
               panelGroupLengthPx * prevElContext.defaultLengthFraction
           : panelGroupLengthPx - nextElLength,
+        prevElContext.initialMinLengthPx,
+        nextElContext.initialMinLengthPx,
       );
     } else if (
-      buttonId.includes(String(this.listPanelContext.size - 1)) &&
+      buttonId.includes(String(this.listPanelContext.size - 2)) &&
```

Comment:
I'm surprised to see a `-2` here. Is this right?

Reply:
That's right because we start the panel id from 0 and the total number of resize handlers is consistently one less than the total number of panels, so the id of the last resize handler is equal to the total number of panels minus 2.

Reply:
I think these could benefit from extraction as named conditionals, like what is it describing? `isFirstButton` and `isLastButton`, for instance. It makes it easier to understand the various cases at high level.